rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/RecoveryCoordinator.java

```diff
@@ -61,15 +64,25 @@
 @ApplicationScoped
 @Path(COORDINATOR_PATH_NAME + "/" + RECOVERY_COORDINATOR_PATH_NAME)
 @Tag(name = "LRA Recovery")
-public class RecoveryCoordinator {
+public class RecoveryCoordinator extends Application {
```

Comment:
It can be either, historically the resource had always extended Application. I moved the initialisation into the resource because Undertow didn't work with a separate Application class.

Reply:
My way of writing the app would be to have only one `Application` but when there is some desigg reason to split it I would understand it.

Reply:
I don't see a problem with having two context roots (the coordinator and the test class LRAParticipant extends Application).